packages/azure-spec-dashboard/src/apis.ts

```diff
@@ -58,6 +59,32 @@ export function getManifestClient() {
   return manifestClient;
 }
 
+export function getCoverageForMode(
+  generatorReports: { [mode: string]: Record<GeneratorNames, ResolvedCoverageReport | undefined> },
+  key: string,
+  mode: string,
+): CoverageReport {
+  if (mode === "standard") {
+    if (
+      (generatorReports["azure"] as any)[key][0]["scenariosMetadata"].packageName ===
+      "@azure-tools/azure-http-specs"
+    ) {
+      return (generatorReports["azure"] as any)[key][1];
+    } else {
+      return (generatorReports["azure"] as any)[key][0];
+    }
+  } else {
+    if (
+      (generatorReports["azure"] as any)[key][0]["scenariosMetadata"].packageName ===
+      "@azure-tools/azure-http-specs"
+    ) {
+      return (generatorReports["azure"] as any)[key][0];
+    } else {
+      return (generatorReports["azure"] as any)[key][1];
+    }
+  }
```

Comment:
I can't tell from looking at this why you're accessing different indices based on these conditions. Can you add some comments so it's clearer what is being returned in each case?

Reply:
@chrisradek Refactored the code and added comment